### PR TITLE
Init EnergyServices Mgmt SDK

### PR DESCRIPTION
**Pull request by fengzhou-msft**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/openenergyplatform/Azure.ResourceManager.EnergyServices/src/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("EnergyServices")]
```
Comment (Yao725) [inline sdk/openenergyplatform/Azure.ResourceManager.EnergyServices/src/Properties/AssemblyInfo.cs:6]: Should be `Microsoft.EnergyServices`

File: sdk/openenergyplatform/Azure.ResourceManager.EnergyServices/src/autorest.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  Ipsec: IPsec|ipsec
+  SSO: Sso
+  URI: Uri
+  Etag: ETag|etag
+
+rename-mapping:
+  DataPartitionAddOrRemoveRequest: DataPartitionAddOrRemoveContent
+  CheckNameAvailabilityRequest: EnergyServiceNameAvailabilityContent
+  CheckNameAvailabilityReason: EnergyServiceNameUnavailableReason
+  CheckNameAvailabilityResponse: EnergyServiceNameAvailabilityResponse
```
Comment (Yao725) [inline sdk/openenergyplatform/Azure.ResourceManager.EnergyServices/src/autorest.md:55]: `EnergyServiceNameAvailabilityResult`?

**Review by Yao725 — COMMENTED** [review]
I don't see the ci.yml, did we miss it?

File: sdk/openenergyplatform/Azure.ResourceManager.EnergyServices/src/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (fengzhou-msft) [reply sdk/openenergyplatform/Azure.ResourceManager.EnergyServices/src/Properties/AssemblyInfo.cs]: Updated

File: sdk/openenergyplatform/Azure.ResourceManager.EnergyServices/src/autorest.md (hunk 2 above)
Comment (fengzhou-msft) [reply sdk/openenergyplatform/Azure.ResourceManager.EnergyServices/src/autorest.md]: Updated

**Review by Yao725 — APPROVED** [review]
LGTM

**Pull request merged**